Tap5050Buyer/Pages/InviteFriendsPage.xaml.cs

```diff
@@ -6,25 +6,153 @@ namespace Tap5050Buyer
 {
     public partial class InviteFriendsPage : ContentPage
     {
+		// Facebook and Twiiter account info
+		//readonly string facebookAppID = "483558705129715"; //Terence
+		readonly string facebookAppID = "838514282900661"; //Chris
```

Comment:
If you use default access modifier the string will be internal then instead of using readonly string it is better to use const string